docs/modules/ROOT/pages/guides/how-to-dynamic-client-registration.adoc

```diff
@@ -10,6 +10,7 @@ Spring Authorization Server implements the https://openid.net/specs/openid-conne
 * xref:guides/how-to-dynamic-client-registration.adoc#configure-client-registrar[Configure client registrar]
 * xref:guides/how-to-dynamic-client-registration.adoc#obtain-initial-access-token[Obtain initial access token]
 * xref:guides/how-to-dynamic-client-registration.adoc#register-client[Register a client]
+* xref:guides/how-to-dynamic-client-registration.adoc#customize-metadata[Customize client metadata]
```

Comment:
This section repeats `SecurityConfig.java` and `ClientRegistrar.java`.
I think we should remove this section and integrate the custom client metadata content into the existing sections to reduce the repeated content.

Reply:
That sounds good. I was struggling a bit with keeping them separate and at the same time demonstrating different configuration approaches. Condensing these together will make this much easier to demonstrate.